### PR TITLE
hooks: fix links to emqx_types

### PR DESCRIPTION
**Pull request by rbino**

### CODE REVIEW

**Review by k32 — APPROVED** [review]

**Comment by k32** [comment]
Thanks for the PR!

File: en_US/advanced/hooks.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ emqx:unhook(Name, {Module, Function}).
 ## Callback function
 The input parameters and returned value of the callback function are shown in the following table:
 
-(For parameter data structure, see:[emqx_types.erl](https://github.com/emqx/emqx/blob/master/src/emqx_types.erl))
```
Comment (zmstone) [inline en_US/advanced/hooks.md:167]: should have changed `master` to `main-v4.3` instead.

**Comment by zmstone** [comment]
https://github.com/emqx/emqx-docs/pull/878

File: en_US/advanced/hooks.md (hunk 1 above)
Comment (zmstone) [reply en_US/advanced/hooks.md:167]: https://github.com/emqx/emqx-docs/pull/878

**Pull request merged**